app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
          Google Maps Android API v2, but you must specify either coarse or fine
          location permissions for the 'MyLocation' functionality.
     -->
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
You are right! I know older version of androids required to have this permission in order to give Android app access to the internet however since Android 7.0 we are no longer requried to have this line as this is now implied by default. Unless your app support older versions as well (below 7.0) then yes you will be required to have it but in our case the minimum version we support is 7.0.